src/pm/hydra/proxy/pmip_pmi_v2.c

```diff
@@ -199,11 +199,6 @@ static HYD_status fn_fullinit(int fd, char *args[])
         HYD_STRING_STASH(stash, MPL_strdup(";spawner-jobid="), status);
         HYD_STRING_STASH(stash, MPL_strdup(HYD_pmcd_pmip.local.spawner_kvsname), status);
     }
-    if (HYD_pmcd_pmip.user_global.debug) {
-        HYD_STRING_STASH(stash, MPL_strdup(";debugged=TRUE;pmiverbose=TRUE"), status);
-    } else {
-        HYD_STRING_STASH(stash, MPL_strdup(";debugged=FALSE;pmiverbose=FALSE"), status);
-    }
```

Comment:
A few background:
* `mpiexec -v` does not enable client debug message with PMI-1, it does that with PMI-2. Thus it is an inconsistent behavior.
* Environment variable `PMI_DEBUG` turns on client debugging. The PMI-2's passing verbose setting via `init_ack` is an extra mechanism. `mpiexec` can simply set and pass on `PMI_DEBUG` to get the same behavior -- i.e. `mpiexec -v` to turn on client debugging.
* I think we should have consistent mechanism/behavior regardless PMI-1 vs PMI-2. If we agree, we can decide which behavior
    * `mpiexec -v` to turn on both  server debugging and client debugging. This is the old PMI-2 behavior.
    * Or `mpiexec -v` to turn on server debugging and use `PMI_DEBUG` to turn on client debugging. This is the old PMI-1 behavior.

Personally I like to have separate control for hydra and client debugging since we often only focused on debug one side at a time. But If we decide for the old PMI-2 behavior, it is easy to change.

Reply:
OK, thanks for the context. I don't have strong feelings about the PMI-2 behavior, so this change seems OK.